marshmallow/fields.py

```diff
@@ -499,14 +500,14 @@ def __init__(self, cls_or_instance, **kwargs):
         if isinstance(cls_or_instance, type):
             if not issubclass(cls_or_instance, FieldABC):
                 raise ValueError('The type of the list elements '
-                                           'must be a subclass of '
-                                           'marshmallow.base.FieldABC')
+                                 'must be a subclass of '
+                                 'marshmallow.base.FieldABC')
```

Comment:
Unnecessary formatting change.

Making formatting changes to code you didn't functionally modify makes a PR much more difficult to review.

I would recommend removing unnecessary formatting changes and squashing most of these commits since this PR has churned so much.